osgp/platform/pom.xml

```diff
@@ -140,4 +113,82 @@
     <module>osgp-secret-management</module>
```

Comment:
Up till now secret management is only used for smart metering. I'm wondering if we should add this to the smart metering profile?